sensor/common/compliance/nodescan_handler_test.go

```diff
@@ -74,11 +72,12 @@ func (s *NodeScanHandlerTestSuite) TearDownTest() {
 // Exec with: go test -race -count=1 -v -run ^TestNodeScanHandler$ ./sensor/common/compliance
 func (s *NodeScanHandlerTestSuite) TestStopHandler() {
 	nodeScans := make(chan *storage.NodeScanV2)
+	errTest := errors.New("example-stop-error")
 	h := NewNodeScanHandler(nodeScans)
 	s.consumeToCentral(h)
 	// This is a producer that stops the handler after producing the first message and then sends many (29) more messages
 	// The intent here it to test whether the handler can shutdown with no leaks when the context is canceled despite of
```

Comment:
I don't see how "context is canceled" has any meaning for the node scan handler that doesn't deal with contexts. It looks like it's best to rework this comment further.

Reply:
Done in [c7ebe5c](https://github.com/stackrox/stackrox/pull/3800/commits/c7ebe5c3b09352a257070aa2e7eee2498a474c2b)